### PR TITLE
Add new identifier profiles to Coverage.identifier (FHIR-47191)

### PR DESCRIPTION
**Pull request by JayMurdoch**
Fix for [FHIR-47191](https://jira.hl7.org/browse/FHIR-47191)

* Add AU Pensioner Concession Card Number, AU Commonwealth Seniors Health Card Number, AU Health Care Card Number to AU Base Coverage.identifier.
* Change log entry

### CODE REVIEW

File: input/pagecontent/changes.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ To help implementers, only the more significant changes are listed here.
     </ul>
   </li>
   <li>Changed context of extension <a href="StructureDefinition-indigenous-status.html">Australian Indigenous Status</a> to add Person and RelatedPerson.</li>
+  <li>Added AU Pensioner Concession Card Number, AU Commonwealth Seniors Health Card Number and AU Health Care Card Number to Coverage.identifier in <a href="StructureDefinition-au-coverage.html">AU Base Coverage</a> (<a href="https://jira.hl7.org/browse/FHIR-47191">FHIR-47191</a>).</li>
```
Comment (dbojicic) [inline input/pagecontent/changes.md:96]: Suggest change to align wording with similar entries: 
Changed Coverage.identifer type in AU Base Coverage to add AU Pensioner Concession Card Number, AU Commonwealth Seniors Health Card Number and AU Health Care Card Number (<a href="https://jira.hl7.org/browse/FHIR-47191">FHIR-47191</a>).

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see comments.

File: input/pagecontent/changes.md (hunk 1 above)
Comment (JayMurdoch) [reply input/pagecontent/changes.md]: Amended, thanks.

Comment (dbojicic) [reply input/pagecontent/changes.md]: Confirmed

**Review by dbojicic — APPROVED** [review]

**Review by brettesler-ext — APPROVED** [review]

**Pull request merged**